sw/device/tests/entropy_src_kat_test.c

```diff
@@ -50,6 +52,49 @@ static void entropy_with_fw_override_enable(dif_entropy_src_t *entropy_src) {
       dif_entropy_src_configure(entropy_src, config, kDifToggleEnabled));
 }
 
+/**
+ * Cleanly disables the SHA3 conditioner while in SHA3 mode, prompting
+ * the release of a conditioned seed.
+ *
+ * If stopping the conditioner fails, due to pending data keep trying for
+ * at most KAT_TEST_TIMEOUT_ATTEMPTS.
+ *
+ * @param entropy An entropy source instance.
+ */
+
+static void stop_sha3_conditioner(dif_entropy_src_t *entropy_src) {
```

Comment:
sorry this is a bit late, but i think we probably should move this to `entropy_src_testutils` eventually.